src/components/Calculator.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+const buttons = [
+  {
+    text: 'AC',
+  },
+  {
+    text: '+/-',
+  },
+  {
+    text: '%',
+  },
+  {
+    text: '÷',
+    className: 'bg-orange',
+  },
+  {
+    text: '7',
+  },
+  {
+    text: '8',
+  },
+  {
+    text: '9',
+  },
+  {
+    text: 'x',
+    className: 'bg-orange',
+  },
+  {
+    text: '4',
+  },
+  {
+    text: '5',
+  },
+  {
+    text: '6',
+  },
+  {
+    text: '-',
+    className: 'bg-orange',
+  },
+  {
+    text: '1',
+  },
+  {
+    text: '2',
+  },
+  {
+    text: '3',
+  },
+  {
+    text: '+',
+    className: 'bg-orange',
+  },
+  {
+    text: '0',
+    className: 'span-2-col',
+  },
+  {
+    text: '.',
+  },
+  {
+    text: '=',
+    className: 'bg-orange',
+  },
+];
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please try to avoid unnecessary disabled eslint rules

Reply:
The linter will warn to convert to functional component without it.